src/main/resources/META-INF/plugin.xml

```diff
@@ -2,7 +2,7 @@
   <id>com.redhat.devtools.intellij.tekton</id>
   <name>Tekton Pipelines by Red Hat</name>
   <version>1.0</version>
-  <vendor email="support@redhat.com" url="http://www.redhat.com">Red Hat</vendor>
+  <vendor email="support@redhat.com" url="http://www.redhat.com">Red-Hat</vendor>
```

Comment:
Are you sure it is Red-Hat? In the OpenShift plugin it is written as Red Hat

Reply:
Yes because the new organization stuff in the markeplace does not allow organizations with space in the ID. Once we validate this feature with this plugin, we will update other plugins